Update RUN_WITH_EXTENSION.md (#86)
OfficeDev/Office-Samples: Excel-HelloWorld-TaskPane-JS/RUN_WITH_EXTENSION.md

# Run the add-in with Office Add-ins Development Kit extension

> We recommend you try this sample by using the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger). The Office Add-ins Development Kit is an end-to end developer tool for building Office add-ins. It helps create, run, and debug an Office Add-in.

Reviewer: ```suggestion
We recommend you try this sample by using the [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger). The Office Add-ins Development Kit is an end-to end developer tool for building Office Add-ins. It helps create, run, and debug an Office Add-in.
```

    If you still have problems, check [troubleshoot development errors]( https://learn.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-development-errors) or [Create an issue](https://aka.ms/officedevkitnewissue) and we'll help you out.  

> # <br>Explore the add-in code
> To explore an Office add-in project, you can start with the key files listed below:

Reviewer: There should only be one H1 per file. Also, if the sentence before a list is a complete sentence, it should end with a period, not a colon.
```suggestion
## Explore the add-in code
To explore an Office add-in project, you can start with the key files listed below.
```

> - The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting `Validate Manifest File` in the `Office Add-ins Development Kit` extension tree view.

Reviewer: I don't think the term "tree view" is a well understood UI construct. I'd recommend avoiding it and just referring to options in the extension. Also, UI buttons should be **bold**, not `code`.
```suggestion
- The `./manifest.xml` file in the root directory of the project defines the settings and capabilities of the add-in.  <br>You can check whether your manifest file is valid by selecting **Validate Manifest File** option from the Office Add-ins Development Kit.
```

- The `./src/taskpane/taskpane.html` file contains the HTML markup for the task pane.
- The `./src/taskpane/taskpane.css` file contains the CSS that's applied to content in the task pane.
- The `./src/taskpane/taskpane.js` file contains the Office JavaScript API code that facilitates interaction between the task pane and the Excel application.

# See also

> * **Explore more add-in samples:** `View Samples` in `Office Add-ins Development Kit` tree view.

Reviewer: ```suggestion
* **Explore more add-in samples:** Select **View Samples** in the Office Add-ins Development Kit.
```

* **Read the documentation:** [Office add-ins documentation](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins)
* **Experience any problems?** [Create an issue](https://aka.ms/officedevkitnewissue) and we'll help you out.

> * **Engage with the team to learn more about updates:** [Join the Microsoft Office Add-ins community call.](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins-community-call)

Reviewer: Slight style thing but avoid having periods in links.
```suggestion
* **Engage with the team to learn more about updates:** [Join the Microsoft Office Add-ins community call](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins-community-call).
```
